test/dotnet.Tests/ParserTests/MigrateParserTests.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using FluentAssertions;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Tools.Migrate;
+using Xunit;
+using Xunit.Abstractions;
+using Parser = Microsoft.DotNet.Cli.Parser;
+
+namespace Microsoft.DotNet.Tests.ParserTests
+{
+    public class MigrateParserTests
+    {
+        public MigrateParserTests(ITestOutputHelper output)
+        {
+            this.output = output;
+        }
+
+        private readonly ITestOutputHelper output;
+
+        [Fact]
+        public void MigrateParserConstructMigrateCommandWithoutError()
```

Comment:
Do you mean we should 
a. set up or reuse real xproj project to run against and make it more end to end
or
b.  just replace the command "dotnet migrate -x 'xproj'" to "dotnet migrate --skip-backup -s -x  ......."

if don't like *a* tho, end to end tests are slow